### PR TITLE
Try to add minimal documentation for all core.stdc bindings

### PR DESCRIPTION
**Pull request by schveiguy**
See discussion here: http://forum.dlang.org/post/m8lsit$1eqi$1@digitalmars.com

### CODE REVIEW

**Comment by schveiguy** [comment]
Note, I did test and make sure all the links are correct, but I did not test how ddoc or ddox outputs this.

File: src/core/stdc/complex.d [hunk 1]
```diff
@@ -1,6 +1,11 @@
 /**
  * D header file for C99.
  *
+ * This contains bindings to selected types and functions from the standard C
+ * header $(LINK2 http://pubs.opengroup.org/onlinepubs/009695399/basedefs/complex.h.html, <complex.h>). Note
+ * that this is not automatically generated, and may omit some types/functions
+ * from the original C header.
```
Comment (andralex) [inline src/core/stdc/complex.d]: Looks good! This seems to be repeated (modulo link), so could you please put it in a macro that we can update wholesale?

````
C_HEADER_DESCRIPTION = This module contains bindings to selected types and functions from the standard C header $(HTTP pubs.opengroup.org/onlinepubs/009695399/basedefs/$1.html, $(LT)$1$(GT)). Note that this is not automatically generated, and may omit some types/functions from the original C header.
```

Plant this bad boy in html.ddoc and expand it in all these modules.

**Comment by andralex** [comment]
Thanks for this work, Steve!

File: src/core/stdc/complex.d (hunk 1 above)
Comment (schveiguy) [inline src/core/stdc/complex.d]: Yep, will do. But I think due to the possible instability of the Internet, I'll take both name of header and URL as parameters.

**Comment by schveiguy** [comment]
BTW, it may be a good idea before pulling to actually build the docs and look at the preview to make sure it's not horrible :)

**Comment by schveiguy** [comment]
Don't pull until I get the html.ddoc update into dlang.org

**Comment by jacob-carlborg** [comment]
Some of the comments in the first two files are not aligned with the declaration, but that might not be your fault. The declarations are not really consistently aligned.

**Comment by schveiguy** [comment]
Anyone with dlang.org commit rights: https://github.com/D-Programming-Language/dlang.org/pull/752

**Comment by schveiguy** [comment]
> Some of the comments in the first two files are not aligned with the declaration, but that might not be your fault. The declarations are not really consistently aligned.

Yeah, it's a combination of the existing file and the editor I use. It's aligning the comment to the previous indentation. This was just a hell of a lot of applying vim macros to add those triple-slashes :) I didn't pay too much attention to the looks.

**Comment by MartinNowak** [comment]
Nice try, still missing from the build system though.

**Pull request merged**